CONTRIBUTING.md

```diff
@@ -160,42 +178,87 @@ Read our {name and link to your style guide} to understand our guidelines for wr
 
 ## Contribution workflow
 
-### Fork and clone repositories
+Official Github Docs on [Forking a repository][forking]
+
+### Initial Setup
 
-To contribute, first fork the repository to your own GitHub account, then clone your fork to your local machine.
+**Fork the Repository on GitHub**
 
-Follow this guide for step-by-step instructions:
-[Fork a repository][forking]
+- Go to the main project repository (`github.com/bettergovph/bettergov`).
+- Click the **"Fork"** button to create a copy under your Github account.
+
+**Clone the Forked Repository to Local Machine**
+
+```sh
+git clone https://github.com/<your-username>/bettergov-fork.git
+cd bettergov-fork
```

Comment:
most of them will most likely click the fork button without making changes to the repository name


```suggestion
git clone https://github.com/<your-username>/bettergov.git
cd bettergov
```